caiman/source_extraction/cnmf/utilities.py

```diff
@@ -33,6 +33,7 @@
 import scipy
 from scipy.sparse import spdiags, issparse, csc_matrix, csr_matrix
 import scipy.ndimage.morphology as morph
+from nd2reader import ND2Reader
```

Comment:
you'd want to make this an optional import, users who don't have nd2 files shouldn't have to install it, something like:

```python

try:
    from nd2reader import ND2Reader
except ImportError:
    HAS_ND2 = False
else:
    HAS_ND2 = True
```

Reply:
It's a pure-python dependency available in conda-forge and pypi; having it as a static dependency is fine (preferable actually)